common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst

```diff
@@ -358,6 +358,30 @@ but the most common are:
 | 25         | Set Home (home location coordinates have been capture)                                             |
 +------------+----------------------------------------------------------------------------------------------------+
 
+
+**GPA: (Global Position Accuracy)**
+
++------------+----------------------------------------------------------------------------------------------------+
+| FIELD      | DESCRIPTION                                                                                        |
++------------+----------------------------------------------------------------------------------------------------+
+| TimeUS     | The GPS reported time since epoch in milliseconds                                                  |
++------------+----------------------------------------------------------------------------------------------------+
+| VDop       | Vertical dilution of precision, a unitless measure of precision                                    |
+|            | https://en.wikipedia.org/wiki/dilution_of_precision                                                |
++------------+----------------------------------------------------------------------------------------------------+
+| HAcc       | The horizontal accuracy as reported by the GPS module, in meters                                   |
```

Comment:
This should always be in meters, but the accuracy measure (CEP, RMS) used depends a lot on the specific GPS manufacturer. It's possible that certain less commonly used drivers get this wrong, but I think the comment that is here is correct.